.gitignore

```diff
@@ -13,7 +13,8 @@ collectedLogs.zip
 .bloop/
 .metals/
 .swp
-project/metals.sbt
+metals.sbt
```

Comment:
has this started building with metals? or supporting various build tools?

Reply:
I'm using VSC with the Metals plugin (alongside IntelliJ) as IDE and this creates these `metals.sbt` files.